plugin.rb

```diff
@@ -65,4 +65,12 @@ module ::DiscourseAi
   end
 
   reloadable_patch { |plugin| Guardian.prepend DiscourseAi::GuardianExtensions }
+
+  register_modifier(:post_with_secure_uploads?) do |_, _, topic|
+    if SharedAiConversation.exists?(target: topic)
+      false
+    else
+      nil
```

Comment:
I'm on the fence on whether the explicit `nil` makes it more or less clear what's going on. I'm leaning on less. It's a bit of a "wat?" factor. 🙂 To me it's clearer that this is a modifier if the "if" stands by itself.

Reply:
Ah yes true, for some reason I was thinking that this was necessary :+1: